test/Test/Cardano/Prelude/GHC/Heap/NormalForm/Classy.hs

```diff
@@ -127,8 +128,8 @@ instance FromModel Int where
   fromModel (IntValue n) k = case n of I# result -> k (I# result)
 
   genModel = Gen.choice [
-        IntThunk <$> genModel
-      , IntValue <$> Gen.int Range.linearBounded
+        IntValue <$> Gen.int Range.linearBounded
+      , IntThunk <$> genModel
```

Comment:
Unnecessary change.

Reply:
The reason for this change (and the similar ones below) is that `Gen.choice` shrinks toward the beginning of the list.

Earlier when I had test failures the shrinker was blowing up because it was building larger and larger trees of `IntThunk`.

So I rearranged all arguments to `Gen.choice` to put the smallest tree at the front.

Reply:
Aha! Gotcha. Makes perfect sense :) Thanks :)